packages/mesos-modules/build

```diff
@@ -5,11 +5,19 @@ popd
 
 source /opt/mesosphere/environment.export
 
+# Add the 3rdparty directories.
+export CPPFLAGS='-I/opt/mesosphere/lib/mesos/3rdparty/include'
+export CFLAGS=-I/opt/mesosphere/lib/mesos/3rdparty/include
+export LDFLAGS='-L/opt/mesosphere/lib/mesos/3rdparty/lib'
+
 mkdir -p build
 pushd build
 /pkg/src/mesos-modules/configure \
      --with-mesos=/opt/mesosphere/active/mesos \
+     --with-openssl=/opt/mesosphere/active/openssl \
```

Comment:
Probably my lack of knowledge. These were included in 1.9 so added it here. @kaysoky can you shed some light as to which module is using openssl? In DC/OS the overlay modules don't necessarily need it, apart from when the master agent comm needs to happen over https but that I am assuming is a libprocess construct and `libmesos` should already be compiled with `openssl`.